students/ericstreit/session02/print_grid.py

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
you seem to have the same thing in "session02" and "lesson02" folders.

sorry for the inconsistent terminology -- but you can use "session" or "lesson", but no need for both :-)